test/test_web.py

```diff
@@ -57,22 +58,48 @@ class AbstractWebTest(unittest.TestCase):
     driver: webdriver.Firefox
 
     def setUp(self) -> None:
+        if not self.share_selenium_web_driver():
+            opts = selenium.webdriver.firefox.options.Options()
+            opts.add_argument("-headless")
+            self.driver = webdriver.Firefox(options=opts)
+
         self.server_runner = InterfaceThreadRunner(shc.web.WebServer, "localhost", 42080, 'index')
         self.server = self.server_runner.interface
 
     @classmethod
     def setUpClass(cls) -> None:
-        opts = selenium.webdriver.firefox.options.Options()
-        opts.add_argument("-headless")
-        cls.driver = webdriver.Firefox(options=opts)
+        if cls.share_selenium_web_driver():
+            opts = selenium.webdriver.firefox.options.Options()
+            opts.add_argument("-headless")
+            cls.driver = webdriver.Firefox(options=opts)
 
     def tearDown(self) -> None:
+        if not self.share_selenium_web_driver():
+            self.driver.close()
+            self.driver.quit()
+
         self.server_runner.stop()
 
     @classmethod
     def tearDownClass(cls) -> None:
-        cls.driver.close()
-        cls.driver.quit()
+        if cls.share_selenium_web_driver():
+            cls.driver.close()
+            cls.driver.quit()
+
+    @staticmethod
+    def share_selenium_web_driver() -> bool:
+        """Should the selenium driver be shared or not.
+
+        Checks whether the environment variable SHARE_SELENIUM_WEB_DRIVER is set.
+        On WSL default ist not sharing the driver since this causes concurrency conflicts.
+        """
+        if (value := os.getenv('SHARE_SELENIUM_WEB_DRIVER')) is not None:
```

Comment:
I'd suggest to prepend "SHC_TEST_" to the environment variable name. We already have the env variable `SHC_TEST_MSQL_URL` for controlling the MySQL interface tests and I think it would be good to have them all in a common "namespace".

I'd actually rename the variable to `SHC_TEST_REUSE_WEB_DRIVER`, but I don't have strong opinions about the precise name.

Reply:
Totally fine w/ your change requests.  Done.